src/Psalm/Report/ConsoleReport.php

```diff
@@ -31,7 +31,7 @@ private function format(\Psalm\Internal\Analyzer\IssueData $issue_data): string
             $issue_string .= 'INFO';
         }
 
-        $issue_reference = $issue_data->link ? ' (see ' . $issue_data->link . ')' : '';
+        $issue_reference = $issue_data->link ? ' - see: ' . $issue_data->link : '';
```

Comment:
Hm, I wonder if the `see:` prefix is even necessary?

Reply:
Square brackets would also be an alternative: `[see ...]`

Reply:
A third option would be to use a gray color for the links to show them less prominent (for environments that don't color links anyway), cf. #4129 (I wonder who's responsible for show the URLs in a nice blue tone)